lib/interface/cli/commands/root/get.cmd.js

```diff
@@ -24,6 +24,10 @@ const get = new Command({
                 choices: ['json', 'yaml', 'wide', 'name', 'id'],
                 group: 'Output Options',
             })
+            .option('pretty', {
```

Comment:
the pretty flag is supported only for the builds entity , so please move this flag to the builds level , not in the get root